### ISSUE TITLE
Missing conditional knockouts from MGI

### ISSUE THREAD

**Issue by mellybelly** [monarch-initiative/dipper#584]
An example is from this paper
http://dx.doi.org/10.1038/ng1874

http://www.informatics.jax.org/allele/reference/J:113097
This should have the combined conditional knockout with phenotype data but does not. How many of these are we missing? making a ticket here to follow up

**Comment by TomConlin** [comment]
Has this been addressed with  https://github.com/monarch-initiative/dipper/pull/595

**Comment by kshefchek** [comment]
I don't think so, but then I'm not exactly sure what our action item is for this.  Is it a curation or ingest issue?  If curation maybe we should move it over to the data-boutique.

**Comment by TomConlin** [comment]
whoops, this is not the ticket I thought I was addressing. will try to move it
